src/pages/dataSetsWip/Edit.tsx

```diff
@@ -42,11 +60,19 @@ export const Component = () => {
     })
     const modelId = useParams().id as string
 
+    const initialValues = dataSetValues.data && {
```

Comment:
I think this should be memoed?

I guess this component are not really re-rendered at this level, however it could cause problems with the `initialValues` if it's re-rendered and not memoed.